package ExceptionHandlingInJava;

import java.util.Scanner;

public class FinallyRunner {

	public static void main(String[] args) {
		Scanner scan = null;
		
		try {
			
			scan = new Scanner(System.in);
			
			int[] numbers = {12,2,3,45};
			
			int number = numbers[6];	
		}
		

		
		System.out.println("Just before closing out main");
	}

}
